src/assets/action.scss

```diff
@@ -24,7 +24,7 @@
 @mixin action-active {
 	li {
 		&.active {
-			box-shadow: inset 4px 0 var(--color-primary);
```

Comment:
please only do the POpover change, keep things separate otherwise it's complcated to review. If you plan to ask for a design change, this should be done in another place :)